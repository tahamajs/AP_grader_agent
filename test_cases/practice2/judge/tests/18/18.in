reserve_table 4 1
reserve_table 1 4
enter 24
reserve_table 24
exit 1
exit 4
exit 24
reserve_table 5 1
reserve_table 1 6
enter 32
reserve_table 32
exit 1
exit 5
exit 32
