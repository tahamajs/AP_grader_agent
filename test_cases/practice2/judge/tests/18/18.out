Mehrad sits at table 1
Paria sits at table 4
Table 2: 4 0
Table 6: 3 0
Table 1: 3 0
Table 4: 1 0
Table 5: 3 0
Table 3: 2 0
Sara sits at table 2
Paria exits!
Mehrad exits!
Sara exits!
Fereshte sits at table 1
Paria sits at table 6
Table 6: 2 0
Table 2: 4 0
Table 5: 3 0
Table 1: 3 0
Table 4: 2 0
Table 3: 2 0
Hasti sits at table 6
Paria exits!
Fereshte exits!
Hasti exits!
